docs/UserGuide.md

```diff
@@ -590,7 +598,20 @@ Action                    | Format
 **List Archived Persons**       | `person /listarchive`                                                           | `p /lar`
 **Unarchive**             | `person (INDEX) /unarchive`                                                   | `p (INDEX) /uar`
 
+<div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
+* For the **Add** and **Remove** commands, please make sure that the `GROUP_NAME` is typed exactly as how it is spelt on
+  the card.
+</div>
+
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+* For the **Create** and **List** commands, if you want to automatically add them to a group, please use the **List** command to make sure the `Group` you want to
+  use the command on is displayed before using them via `GROUP_INDEX`.
+* For the **List** commands, to view list of persons in a particular subgroup, you should first use **List** command to list out all the subgroups in a `Group` you want to view,
+  then list out the persons inside that subgroup by **List** command with the `GROUP_INDEX` you want to list out.
+</div>
+
 ### Group
+Refer to the details of the group commands [here](#group).
```

Comment:
This goes to here literally. It goes to the summary. I think it is suppose to go up to the commands.